guides/common/modules/proc_tuning-apache-httpd-child-processes.adoc

```diff
@@ -0,0 +1,27 @@
+[id="tuning_apache_httpd_child_processes_{context}"]
+= Tuning Apache httpd child processes
+
+By default, HTTPD uses event request handling mechanism.
+When the number of requests to httpd exceeds the maximum number of child processes that can be launched to handle the incoming connections, httpd raises an HTTP 503 Service Unavailable error.
+Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failures on your {Project} services side due to the dependency of components like Pulp on the availability of httpd processes.
```

Comment:
Pulp is Katello-only. I'm wondering if it can be rewritten in a way that is generic, or if we should just hide the sentence in non-Katello builds.

Reply:
Yes, I do agree on this point. Change has been done.

Reply:
```suggestion
Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failures on your {Project} services side due to the dependency of some components on the availability of httpd processes.
```
@Imaanpreet We can go with this wording if there are any non-Katello components that depend on it. Otherwise, we would have to render this whole module only for products that contain Katello.

Reply:
Oh, I didn't notice that we already render it only for katello,satellite,orcharhino. So we can actually resolve this thread.

Reply:
okay @Lennonka : let's resolve this.